cfme/tests/test_utilization_metrics.py

```diff
@@ -251,8 +251,8 @@ def test_raw_metric_host_network(metrics_collection, appliance, provider):
 @pytest.mark.uncollectif(
     lambda provider: provider.one_of(CloudProvider))
 @pytest.mark.meta(
-    blockers=[BZ(1424589, forced_streams=["5.7", "5.8", "upstream"],
-        unblock=lambda provider: not provider.one_of(RHEVMProvider))]
+    blockers=[BZ(1424589, forced_streams=["5.8", "upstream"],
+    unblock=lambda provider: not provider.one_of(RHEVMProvider))]
 )
```

Comment:
@digitronik , test_raw_metric_host_disk needs to be skipped for RHV in 59 as well.